### ISSUE TITLE
Flake in AccessibilityBridgeTest.canFireChildrenChangedCorrectly

### ISSUE THREAD

**Issue by zanderso** [flutter/flutter#75284]
From https://ci.chromium.org/ui/p/flutter/builders/prod/Mac%20Host%20Engine/8786/overview

```
[ RUN      ] AccessibilityBridgeTest.canFireChildrenChangedCorrectly
../../flutter/shell/platform/common/cpp/accessibility_bridge_unittests.cc:153: Failure
Expected: (actual_event.find(ui::AXEventGenerator::Event::CHILDREN_CHANGED)) != (actual_event.end()), actual: 8-byte object <38-67 2C-EC FE-7F 00-00> vs 8-byte object <38-67 2C-EC FE-7F 00-00>
[  FAILED  ] AccessibilityBridgeTest.canFireChildrenChangedCorrectly (2 ms)
```

on https://github.com/flutter/engine/commit/84bb317a13ceb08e2393d74e936b101e9cfb1c78, but looks unrelated.

**Comment by chinmaygarde** [comment]
cc @chunhtai @gw280

**Comment by chunhtai** [comment]
This is quite weird, i can't think of any possible way this can fail. I cannot reproduce it locally neither. Does this happen a lot?

**Comment by chinmaygarde** [comment]
Tests are run in random order with the seed printed before each run. I'd try to run the test with same seed and repeat it as well. In the case of the failing run, it'd be `--gtest_shuddle --gtest_random_seed= 26530 --gtest_repeat=-1`. Runs can be attempted with various sanitizers (go/flutter-engine-sanitizers).

I haven't seen this often and the priority reflects that. But this code is also new so it may be that it just hasn't come up often enough.

**Comment by zanderso** [comment]
From engine flake triage: This has not flaked recently, so I am going to close this issue as stale.
